app/models/course/assessment/question/text_response.rb

```diff
@@ -0,0 +1,12 @@
+class Course::Assessment::Question::TextResponse < ActiveRecord::Base
+  acts_as :question, class_name: Course::Assessment::Question.name, inverse_of: :actable
+
+  has_many :solutions, class_name: Course::Assessment::Question::TextResponseSolution.name,
+           dependent: :destroy, foreign_key: :question_id, inverse_of: :question
```

Comment:
Actually not sure, I was following what `Course::Assessment::Question::MultipleResponse` does